.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: python
 python:
   - "3.6"
+  - "3.11"
```

Comment:
@Alopalao, this `.travis.yml` is no longer used, let's take the opportunity to also nuke it. It won't be a blocker for this PR, but let's nuke it either on this PR or a subsequent one.

Reply:
Deleted